src/types/User.tsx

```diff
@@ -0,0 +1,80 @@
+export interface User {
+  id: number,
```

Comment:
Consider using more descriptive names for properties. For example, `id` could be `userId`, `name` could be `fullName`, `username` could be `userName`. 🤔